php/templates/configmap-fpm.yaml

```diff
@@ -1,3 +1,22 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "php.fullname" . }}-fpm-phpini
+  labels:
+    app: {{ template "php.name" . }}
+    chart: {{ template "php.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    checksum/fpm-phpini: {{ tpl (index .Values.fpm.templates "php.ini" | toYaml) . | sha256sum }}
+{{- with .Values.fpm.annotations }}
+{{ toYaml . | indent 4 }}
+{{- end }}
+data:
+  php.ini:
+{{- tpl (index .Values.fpm.templates "php.ini") . | toYaml | indent 2 }}
+
```

Comment:
@cw-niinuma 
I think that it doesn't need to add the ConfigMap, because you can add extra files (ex. php.ini) with using `extraVolumeMounts` and `templates`. 

Example code is below.

```
fpm:
  extraVolumeMounts:
    - name: php-ini
      subPath: php.ini
      mountPath: /usr/local/etc/php/php.ini

  templates:
    php-ini:
      php.ini: |
        serialize_precision: 17
        expose_php: off
        max_input_vars: 10000
        and more...
```

Reply:
I am sorry for misunderstanding the specification.  Ignore above comment.